parallel-consumer-core/src/main/java/io/confluent/parallelconsumer/ParallelConsumerOptions.java

```diff
@@ -131,6 +131,19 @@
     @Builder.Default
     private final Duration defaultMessageRetryDelay = Duration.ofSeconds(1);
 
+    /**
+     * Controls how long to block while waiting for the {@link Producer#send} to complete for any ProducerRecords
+     * returned from the user-function. Only relevant if using one of the produce-flows and providing a
+     * {@link ParallelConsumerOptions#producer}. If the timeout occurs the record will be re-processed in the user-function.
+     *
+     * Consider aligning the value with the {@link ParallelConsumerOptions#producer}-options to avoid unnecessary re-processing
+     * and duplicates on slow {@link Producer#send} calls.
+     *
+     * @see org.apache.kafka.clients.producer.ProducerConfig#DELIVERY_TIMEOUT_MS_CONFIG
+     */
+    @Builder.Default
+    private final Duration sendTimeout = Duration.ofSeconds(2);
```

Comment:
@astubbs : Should we increase the default timeout? Just kept the original value for first draft.

Reply:
Actually I think yes increase the time out. I hit this issue myself just running the test suite in parallel - slower I guess in some ways. What would you suggest using as the default?

Reply:
Hmm, perhaps align with default producer timeout? 🤔 
https://docs.confluent.io/platform/current/installation/configuration/producer-configs.html#producerconfigs_delivery.timeout.ms